src/App.css

```diff
@@ -717,7 +706,86 @@ h1 .num-contexts {
   margin-left: -17px;
   top: -0.15em;
 }
-*/
+
+/* Android Section */
+
+.android .child > .bullet {
+  left: -2px;
+}
+
+.android .child > .bullet::before {
+  content: '▸'; /* •◦◂◄◀︎ ➤▹▸►◥ */
+  position: relative;
+  margin-left: -16.8px;
+  opacity: 0.8;
+  transition: opacity 0.75s ease-in-out;
+}
+
+.android .child > .bullet::after {
+  top: -1px;
+}
+ 
+.android .child.expanded:not(.leaf) > .bullet,
+.android .child.editing:not(.leaf) > .bullet,
+.android .child.cursor-parent:not(.leaf) > .bullet,
+.android .child.cursor-grandparent:not(.leaf) > .bullet {
+  display: inline-block;
+  margin-right: -7px; /* compensate for inline-block */
+  transform: rotate(90deg);
+  transform-origin: -11px 9px;
+  margin-left: -2.8px;
+  margin-top: 1.5px;
+}
+
+.android .dragging {
+  color: green;
```

Comment:
The different colors are great for debugging, I just request that you change the color back before we merge!

Reply:
Sure thing !!!